packages/client/hmi-client/src/components/widgets/autocomplete.vue

```diff
@@ -1,3 +1,5 @@
+import { computed } from "vue"; import { computed } from "vue";
+
```

Comment:
This should not be there, maybe a pasting mistake.

Reply:
done